scripts/globals/additional_effects.lua

```diff
@@ -120,7 +120,8 @@ xi.additionalEffect.attack = function(attacker, defender, baseAttackDamage, item
         ABSORB_STATUS = 10,
         SELF_BUFF = 11,
         DEATH = 12,
-    }
+		REMOVE_UNKILLABLE = 13, --Solely for Buccaneer's Knife with Brigandish Blade
```

Comment:
Should we really be adding an AE for a single mob?  This seems like something that could more elegantly be handled elsewhere.